internal/testdata/node/docker-compose.autogen-1.yml

```diff
@@ -16,6 +16,20 @@ services:
       - LAGOON_LOCALDEV_HTTP_PORT=3000
       - LAGOON_ROUTE=http://node.docker.amazee.io
 
+  elasticsearch:
+    image: uselagoon/elasticsearch-7:latest
```

Comment:
I'd have a preference in the tests for using an image we still support (ES is EOL for us) - would the Opensearch image work?

Reply:
well these never get deployed anywhere beyond being in the tests. so im going to change it to something completely unrelated if anything

Reply:
fixed